src/Time/Units.hs

```diff
@@ -131,6 +125,13 @@ type Day         = Time DayUnit
 type Week        = Time WeekUnit
 type Fortnight   = Time FortnightUnit
 
+-- | Type-level list that consist of all time.
+type AllTimes =
```

Comment:
I think it's a good idea to add explicit kind to `AllTimes` variable here.

Reply:
It's not possible to write so `AllTimes :: [*]`..

Reply:
Oh, sorry. I never did this and somehow thought it's possible.